py/server/deephaven/parquet.py

```diff
@@ -131,6 +138,7 @@ def read(
     is_refreshing: bool = False,
     file_layout: Optional[ParquetFileLayout] = None,
     table_definition: Union[Dict[str, DType], List[Column], None] = None,
+    special_instructions: Optional[object] = None,
```

Comment:
I don't love having a raw object type here.  It makes me think that we're bleeding jpy java objects, and we have gone to great lengths to avoid this.

@jmao-denver should assess.

Reply:
Hi Chip, I had a discussion with Jianfeng for this and have updated the design. Let me know if this looks better to you.